build.gradle

```diff
@@ -242,6 +242,19 @@ configure(publishProjects) { Project subproject ->
                 }
             }
         }
+
+        if (project.hasProperty('maven.repo.s3') && subproject.name.equals("cordapp-cpk2")){
```

Comment:
So is this going to publish `cordapp-cpk2` to S3 while still publishing all of the other plugins to Artifactory? Perhaps you want:
```gradle
def publishProjects = hasProperty('maven.repo.s3') ? [ project(':cordapp-cpk2') ] : childProjects.values()
```
and _then_ define the `repositories` block.

Reply:
Thanks @chrisr3 updated